kv/kvStore.go

```diff
@@ -0,0 +1,34 @@
+package kv
+
+import (
+	"errors"
+	"fmt"
+)
+
+const MaxMem = 1 << (10 * 3) // 1 GB
+const defPath = "."          // create in local directory
+
+type KeyValueStore interface {
+	Put(a1 uint64, a2 [10]byte)
```

Comment:
I'd say that would be an error.
As mentioned in the exercise pdf (or in the lecture, can't remember exactly) he told us that giving the interface more advanced methods like `upsert()` is fine, but not mandatory.

Therefore it would create a collision in case it'd get implemented later.